Adding SSL VPN section to readme file (#5)
fortinet/terraform-secure-remote-access-beta: README.md

The default admin username and password can be found in vars.tf under `admin_name` and `admin_password`. <br>

> **Note:** For ease of configuration, search for **EasyKey** in the output. It will contain configuration that can be applied to Spoke VPN devices.

# Spoke FortiGate Setup

>   

Reviewer: ```suggestion

```

> <i>Note: <b>EasyKey</b> from the output will contain configuration that can be applied to Spoke VPN devices for ease of configuration. </i>

Reviewer: ```suggestion
> **Note:** For ease of configuration, search for **EasyKey** in the output. It will contain configuration that can be applied to Spoke VPN devices.
```

# Spoke FortiGate Setup
  

> Once the Terraform deployment is complete, follow the steps below to attach the spoke to the FortiGate Hub

Reviewer: ```suggestion
Once the Terraform deployment is complete, follow the steps below to attach the spoke to the FortiGate hub.
```

> ### Authentication:

Reviewer: ```suggestion
### Authentication
```

> 1. Under **Remote IP Address** enter the Public IP address of the FortiGate you deployed. You can find this value in the outputs. You can also run `terraform output` in the deployment folder to see the results again.

Reviewer: ```suggestion
Under **Remote IP Address** enter the Public IP address of the FortiGate you deployed.<br>You can find this value in the outputs. Run `terraform output` in the deployment folder to see the results again.
```

2. The **Outgoing interface** should adjust automatically based on the **Remote IP address** entered.

> 3. Enter the **Pre-shared key**. This can be found in the vars.tf file under `psk_key`.

Reviewer: ```suggestion
3. Enter the **Pre-shared key**. This can be found in the `vars.tf` file under `psk_key`.
```

   ![FortiOS Admin Profile](./imgs/bring_up_phase_selectors.png)

# SSL VPN Users/Groups creation and configuration guide

### Create a new local user
> These steps are performed on the FortiOS GUI.

1. On the navigation bar, select **User & Device > User Definition**.
2. Click **Create New**:

  ![Create New Local User](./imgs/create_new_user.png) 

3. Select **Local User**.
4. Set up credentials for the user.
5. (Optional) Add an **Email address**.
6. Click **Submit**.

### Create a new User Group
> These steps are performed on the FortiOS GUI.

1. On the navigation bar, select **User & Device > User Groups**.
2. Click **Create New**:

  ![Create New User Group](./imgs/user_group_selection.png)

3. Under **Type**, select **Firewall**.
4. Enter the name of the group and select members:

  ![User Group Selection](./imgs/user_group_selection.png)  

5. Click **OK**.

### Add a user/user group to Authentication/Portal Mapping
> These steps are performed on the FortiOS GUI.

1. On the navigation bar, select **VPN > SSL-VPN Settings**.

> 2. At the bottom of **SSL-VPN Settings** page, there is a table to assign **User** and/or **User group** to specific portals to use.

Reviewer: ```suggestion
2. At the bottom of the **SSL-VPN Settings** page, there is a table to assign a **User** and/or **User group** to specific portals.
```

   ![FortiOS Admin Profile](./imgs/bring_up_phase_selectors.png)

> #SSL VPN Users/Groups creation and configuration guide

Reviewer: ## instead of #?

> ### Create a new local user
> > This section will showcase how to create new local user on FortiOS GUI
> 
> 1. Go to User & Device > User Definition from the navbar
> 2. Click `Create New`
> 
>   ![Create New Local User](./imgs/create_new_user.png) 
> 
> 3. Select `Local User`
> 4. Set up Credentials for the user
> 5. Add `Email address` _(optional)
> 6. Click `Submit` to create user

Reviewer: ```suggestion
### Create a new local user
> These steps are performed on the FortiOS GUI.

1. On the navigation bar, select **User & Device > User Definition**.
2. Click **Create New**:

  ![Create New Local User](./imgs/create_new_user.png) 

3. Select **Local User**.
4. Set up credentials for the user.
5. (Optional) Add an **Email address**.
6. Click **Submit**.
```

> ### Create a new User Group
> > This section will showcase how to create a new user group on FortiOS GUI
> 
> 1. Go to User & Device > User Groups from the navbar
> 2. Click `Create New`
> 
>   ![Create New User Group](./imgs/user_group_selection.png)
> 
> 3. Select Type `Firewall`
> 4. Type in the name of the group and select members
> 
>   ![User Group Selection](./imgs/user_group_selection.png)  
> 
> 5. Hit `OK` to create User Group

Reviewer: ```suggestion
### Create a new User Group
> These steps are performed on the FortiOS GUI.

1. On the navigation bar, select **User & Device > User Groups**.
2. Click **Create New**:

  ![Create New User Group](./imgs/user_group_selection.png)

3. Under **Type**, select **Firewall**.
4. Enter the name of the group and select members:

  ![User Group Selection](./imgs/user_group_selection.png)  

5. Click **OK**.
```

> ### Adding User/User Group to SSL VPN Policy
> > This section will showcase how to add User/User Groups to SSL VPN Policy
> 
> 1. Set up a name for the Policy(if not editing). <br>
> 2. Incoming Interface should be **SSL-VPN tunnel interface (ssl.root)**
> 3. Select desired Outgoing interface
> 4. In Sources select addresses and switch to `User` tab to select User and/or User group 
> 5. Select Destination and Service and then hit `OK` to create/edit policy 
> 
>   ![Policy Settings](./imgs/policy_user_selection.png)

Reviewer: ```suggestion
### Adding a User/User Group to the SSL VPN Policy
> These steps are performed on the FortiOS GUI.

1. Enter a **Name** for the policy (if not editing).
2. The **Incoming Interface** should be **SSL-VPN tunnel interface (ssl.root)**.
3. Select the desired **Outgoing interface**.
4. Under **Sources**, select addresses and on the **User** tab select the **User** and/or **User group**. 
5. Select a **Destination** and **Service**.
6. Click **OK**.

  ![Policy Settings](./imgs/policy_user_selection.png)
```
